src/sections/LegalSection.module.scss

```diff
@@ -1,16 +1,13 @@
-$medium: 768px;
-$large: 992px;
+@use "src/theme/mixin";
 
 .legalSubheader {
   margin: 0;
   font-size: 1.4rem;
 }
 
-@media (min-width: $large) {
+@media (min-width: 992px) {
```

Comment:
What's the reason for this change?

Reply:
Because I though it's weird to have breakpoint variables locally in one file, while only using them once.
It probably is best to define global breakpoints similarly to #56.

Reply:
That makes sense, thanks for explaining!